protocol/avdl/keybase1/user.avdl

```diff
@@ -86,38 +112,12 @@ protocol user {
     */
   UserSettings loadMySettings(int sessionID);
 
-  /**
-    The list-tracking functions get verified data from the tracking statements
-    in the user's sigchain.
-
-    If assertion is empty, it will use the current logged in user.
-    */
-  array<UserSummary> listTracking(int sessionID, string filter, string assertion);
-  string listTrackingJSON(int sessionID, string filter, boolean verbose, string assertion);
-
   /**
     Load all the user's public keys (even those in reset key families)
     from the server with no verification
     */
   array<PublicKey> loadAllPublicKeysUnverified(int sessionID, UID uid);
 
-  record UserSummary2 {
```

Comment:
I unified this with UserSummary, but kept it compatible with the UserSummary2 that is in our leveldbs. Removed isFollower/isFollowee fields.